README.md

```diff
@@ -2,28 +2,29 @@
 
 ### Overview
 
-Easy deploy AI models to Kubernetes using LocalAI, vLLM, DeepSpeed-MII, NVIDIA Triton and custom Docker images.
+Deploy AI models and apps to Kubernetes without hitting every pitfall.
```

Comment:
```suggestion
Deploy AI models and apps to Kubernetes without hitting any pitfalls and feeling overwhelmed by the ginormous YAMLs.
```

Reply:
I don't think we can claim they won't hit any pitfalls :-)